### PR TITLE
[COST-3545] Add distributed cost to OpenShift provider map

### PR DESCRIPTION
**Pull request by djnakabaale**
## Jira Ticket

[COST-3545](https://issues.redhat.com/browse/COST-3545)

## Description

This change will add distributed cost to OCP provider map and show a `distributed` cost field in the cost total for the total and for each project in the api response when grouping by project.

## Testing

1. Checkout Branch
2. Restart Koku
3. load test data
4. Hit endpoint 
    ```
    http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/costs/?group_by[project]=*&group_by[cluster]=Test OCP on Premises
    ```

    1. You should see a `distributed` field in the cost total in the api response.
    
    2. If you search for `Platform unallocated`, the value for distributed cost for those projects should have `0.0` value.
    
    ```
        "distributed": {
            "value": 0.0,
            "units": "USD"
        },
    ```

## Notes

...

### CODE REVIEW

File: koku/api/report/ocp/provider_map.py [hunk 1]
```diff
@@ -208,6 +216,10 @@ def __init__(self, provider, report_type):
                             "cost_total": self.cloud_infrastructure_cost_by_project
                             + self.markup_cost_by_project
                             + self.cost_model_cost,
+                            "cost_distributed": self.cloud_infrastructure_cost_by_project
```
Comment (adberglund) [inline koku/api/report/ocp/provider_map.py:219]: What do you think about `cost_total_distributed` ?

Comment (djnakabaale) [reply koku/api/report/ocp/provider_map.py]: it seems more consistent, thanks 😀 
I've updated the provider map

**Comment by djnakabaale** [comment]
smoke tests:  https://ibutsu.apps.ocp4.prod.psi.redhat.com/runs/e2b5c3da-25ec-4518-9855-e5e40870450f

failing smoke test `test_api_ocp_on_gcp_source_raw_calc`, https://ibutsu.apps.ocp4.prod.psi.redhat.com/results/70c584b2-976d-4b1f-9258-5f0172228551#summary

**Review by adberglund — APPROVED** [review]

**Pull request merged**